resources/function_help/json/decode_uri

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "decode_uri",
+  "type": "function",
+  "description": "Takes a layer and decodes the uri of the undelying data provider. It depends on the dataprovider, which data is available.",
+  "arguments": [ {"arg":"layer","description":"The layer for which the uri should be decoded."},
+                 {"arg":"part","description":"The part of the uri to return. If unspecified, a map with all uri parts will be returned."} ],
+  "examples": [ { "expression":"decode_uri(@layer)", "returns":"{'layerId': '0', 'layerName': '', 'path': '/home/qgis/shapefile.shp'}"},
+                { "expression":"decode_uri(@layer)", "returns":"{'layerId': NULL, 'layerName': 'layer', 'path': '/home/qgis/geopackage.shp'}"},
```

Comment:
geopackage.shp ? :upside_down_face:

Reply:
There's something in there for the fanboys and those who want stable, eh?